### ISSUE TITLE
Return the unlimited 5 mins postpone button

### ISSUE THREAD

**Issue by MnemosyneVL** [Stigmatoz/web-activity-time-tracker#112]
First of all, great extension. I use it for a while and I recommend it to everyone.

In the latest update, the "postpone for 5 mins" button has been replaced with a one-time use (per day) button.
It creates a lot of discomfort.
There are several problems that it caused:
1. The following day after the page was blocked by the limiter I used this button (postpone for 5 mins) to go back to the page and continue where I left off. Now I have to copy the link to the blocked page from the address bar.
2. Often one postponement per day is not enough to finish what I wanted, because there are usually several pages from the same domain that are being blocked. So I have to copy the pages to another browser to finish what I was doing on all pages.

Possible solutions:
1. Maybe adding a setting that allows for no postponing, one 5 mins postpone, or infinite postponing will solve the issue.
2. Adding a link to the blocked page on the block screen, so that I could come back to it the next day would also be great to solve the first problem

**Comment by Stigmatoz** [comment]
Thank you for your feedback. It's really important for me.
I will add the source URL that was blocked to the blocked page.
And I'll think of another setting that can delay blocking more than once during the day.

**Comment by tcolonhezi** [comment]
About the postpone button, after the last update, it only showed up once, and after that, never again. I waited for the next day, but still, it didn't show up for me.

**Comment by Stigmatoz** [comment]
This button will appear again 24 hours after you click. And now I am writing this comment and I understand that this is not an entirely obvious behavior. Need to show the button just the next day.

**Comment by scandal43** [comment]
The postpone button still isn't returning for me. I clicked it once last week, and it has not shown up since.
